src/tests/userDashboard/actions/index.test.js

```diff
@@ -60,7 +58,7 @@ describe('Test suite for user dashboard actions', () => {
     ];
 
     const store = mockStore({});
-    return store.dispatch(fetchUserDictionary('emasys')).then(() => {
+    return store.dispatch(fetchDictionariesUser('chriskala')).then(() => {
```

Comment:
The word `fetchDictionariesUser` sounds confusing, is it possible to change it to `fetchUserDictionaries`. And did you need to change it?

Reply:
@hadijahkyampeire Done

Reply:
Yes i had to change it